### PR TITLE
Update for seven_zip install

### PR DESCRIPTION
**Pull request by jaymalasinha**
@chef-cookbooks/engineering-services

### CODE REVIEW

File: recipes/_packaging.rb [hunk 1]
```diff
@@ -65,6 +65,6 @@
   include_recipe 'windows-sdk::windows_sdk'
 
   omnibus_env['PATH'] << node['wix']['home']
-  omnibus_env['PATH'] << node['7-zip']['home']
+  omnibus_env['PATH'] << windows_safe_path_join(ENV['SYSTEMDRIVE'], 'Program Files', '7-zip')
```
Comment (scotthain) [inline recipes/_packaging.rb]: Can we just use `node['seven-zip']['home']` since we set this in `common.rb`?

**Comment by jaymalasinha** [comment]
@scotthain @schisamo updated omnibus_env and tested by successfully converging 
windows-server-2012r2-standard and verifying 7-zip install path and version 15.14

**Comment by schisamo** [comment]
@jaymalasinha I'm looking into the Travis failures on Debian 8 (2 sub-jobs in the matrix). Those two jobs fail when attempting to install Chef from omnitruck. It's something SSL related in the Debian 8 base container. We rolled out changes to Omnitruck yesterday that return packages.chef.io URLs so Debian 8's default config is not liking this new SNI SSL config. The weird thing is Debian 7 works perfectly!

**Comment by yzl** [comment]
:+1: pending travis testing

**Comment by schisamo** [comment]
https://github.com/chef-cookbooks/omnibus/pull/159 has been merged...please rebase this branch onto master.

**Comment by jaymalasinha** [comment]
rebased to master

**Pull request merged**